### DISCUSSION TITLE
I want to get the user's info after user verifies email

### DISCUSSION THREAD

**Discussion by thetejasmagade** [supabase/supabase#23012] [Questions]
I want to get the user's info after user verifies email.
Is there anything something like auth events for email verification? I want to save some basic user's info in my DB but, after successful account creation.

**Comment by GaryAustin1** [comment]
I have seen something like this used on the database side to trigger a function to update your table as you see fit.
```
create or replace trigger on_email_confirmed_at_updated
  after update of email_confirmed_at on auth.users
  for each row execute procedure public.update_email_confirmed_at()
```
On the client side you have a confirmed at property in the user object that will get set.
![image](https://github.com/supabase/supabase/assets/54564956/b443f9c3-b886-4181-bd0b-4a4bc1205cf4)
